purchase/products/reordering.rst

```diff
@@ -1,51 +1,79 @@
-=================================================
-Don’t run out of stock thanks to reordering rules
-=================================================
+============================================
+Don’t run out of stock with reordering rules
+============================================
 
-To make sure you never run out of stock, you might want to define
-*Reordering Rules* on products. Thanks to reordering rules, Odoo will
-help you to replenish your stock when it reaches a minimum stock level.
+To make sure you never run out of stock, you can define *Reordering Rules* on products. Thanks to
+them, Odoo can help you replenish your stock automatically when it reaches set quantities or
+whenever a sales order is created.
 
-Set a reordering rule
-=====================
+.. important:: You need to install the *Inventory app* to have access to reordering rules.
 
-Open a stockable product, you'll find the *Reordering Rules* tab.
+Configure your storable product
+===============================
 
-.. note::
-  You can also access all your reordering rules from
-  :menuselection:`Inventory --> Master Data --> Reordering Rules`.
+Open or create a product with its *Product Type* set to *Storable Product*.
 
-.. image:: media/setup_stock_rule01.png
-  :align: center
+.. image:: media/reordering-storable-product.png
+   :align: center
+   :alt: Set the product type in Odoo
 
-Once in the next menu, you can create the reordering rules linked to
-your product. From now on, every time this product reaches the minimum
-quantity it will replenish your stock to your **maximum quantity**.
+As you are purchasing this product from a vendor, go to the product's *Purchase tab* and add a
+vendor and a price by clicking on *Add a line*. You can add multiple vendors, but make sure to order
+them correctly, since reordering rules always use the first vendor in a list.
 
-.. image:: media/setup_stock_rule02.png
-  :align: center
+.. image:: media/reordering-product-vendor.png
+   :align: center
+   :alt: Add vendor to a product in Odoo
 
 .. tip::
-  The replenishments will take place when the scheduler in the
-  *Inventory* module runs. By default in Odoo, the schedulers will run
-  every night at 12:00PM.
+   By default, a draft purchase order is created. However, if you have enabled *Purchase Agreements*,
```

Comment:
That's 101 chars! :p 
You can safely break in the middle of \*a text between\*.